omnibus/verification/verify.rb

```diff
@@ -194,10 +194,10 @@ def components
 
       add_component "chef-apply" do |c|
         c.gem_base_dir = "chef-apply"
-      #   c.unit_test do
```

Comment:
Can you try uncommenting these and running an ad-hoc build? I found some of our other commented out verifications have started passing recently.